src/Microsoft.HttpRepl.Tests/Commands/DeleteCommandsTests.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.HttpRepl.Commands;
+using Microsoft.HttpRepl.Tests.SampleApi;
+using Microsoft.Repl;
+using Microsoft.Repl.ConsoleHandling;
+using Microsoft.Repl.Parsing;
+using Xunit;
+
+namespace Microsoft.HttpRepl.Tests.Commands
+{
+    public class DeleteCommandTests : BaseHttpCommandTests, IClassFixture<HttpCommandsFixture<DeleteCommandsConfig>>
+    {
+        private readonly DeleteCommandsConfig _config;
+        public DeleteCommandTests(HttpCommandsFixture<DeleteCommandsConfig> deleteCommandsFixture)
+        {
+            _config = deleteCommandsFixture.Config;
+        }
+
+        [Fact]
+        public async Task ExecuteAsync_WithNoBasePath_VerifyError()
+        {
+            await VerifyErrorMessage(command: new DeleteCommand(),
+                                     commandText: "DELETE",
+                                     baseAddress: null,
+                                     path: null,
+                                     expectedErrorMessage: Resources.Strings.Error_NoBasePath);
+        }
+
+        [Fact]
+        public async Task ExecuteAsync_WithMultipartRoute_VerifyOutput()
+        {
+            await VerifyResponse(command: new DeleteCommand(),
+                                 commandText: "DELETE",
+                                 baseAddress: _config.BaseAddress,
+                                 path: "a/file/path.txt",
+                                 expectedResponseLines: 5,
+                                 expectedResponseContent: "File path delete received successfully.");
+        }
+
+        [Fact]
+        public async Task ExecuteAsync_WithOnlyBaseAddress_VerifyOutput()
+        {
+
```

Comment:
nit: empty line. Also can we keep the formatting consistent with above?

Reply:
Done in a29a3df